lab-jason/lib/joke.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+
+function tellJoke(filename, cb){
+  fs.readFile(`./assets/${filename}`, (err, buff) => {
+    if (err) {
+      cb(err);
```

Comment:
If you're going to check if `cb` exists below, you should probably check it here, too. Maybe `throw err` if `cb` doesn't exist?